phalcon/validation/messageinterface.zep

```diff
@@ -70,6 +70,6 @@ interface MessageInterface
 	/**
 	 * Magic __set_state helps to recover messages from serialization
 	 */
-	public static function __set_state(array! message) -> <MessageInterface>;
+	public static function __set_state(array! message);
```

Comment:
This is a factory method

Reply:
i did this because of https://github.com/phalcon/cphalcon/blob/master/phalcon/validation/message/group.zep#L269

i am not sure what is the correct thing to do here, I tough this is the safest way to do it, not to break other apps

Reply:
Group class has its own `__set_state` and Group does not implement MessageInterface